helmfile.d/snippets/defaults.yaml

```diff
@@ -570,7 +570,63 @@ environments:
             enabled: false
             serving:
               replicas: 1
-            _rawValues: {}
+            resources:
+              queueSidecar:
+                requests:
+                  cpu: 25m
+                  memory: 256Mi
```

Comment:
It is quite a lot of memory requested for FaaS

Reply:
The default is `400Mi` if no value is set. I agree that it is a lot. Actual consumption (idle state) is `8Mi`. I lowered it to `64Mi`